### PR TITLE
Code snippet that allows importing cookies from Chrome

### PR DESCRIPTION
**Pull request by OMGhozlan**
Similar to the existing code snippet that facilitates importing cookies from Firefox, this code provides the same functionality except for Chrome. May require path validation for Mac OS systems

### CODE REVIEW

File: docs/codesnippets/import_chrome_session.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# https://n8henrie.com/2013/11/use-chromes-cookies-for-easier-downloading-with-python-requests/
+
+from argparse import ArgumentParser
+from glob import glob
+from os.path import expanduser
+from platform import system
+from sqlite3 import OperationalError, connect
+import win32crypt
```
Comment (aandergr) [inline docs/codesnippets/import_chrome_session.py:8]: What is `win32crypt`?

Comment (OMGhozlan) [reply docs/codesnippets/import_chrome_session.py:8]: It's a module that is used for cryptographic operations. Chrome cookies are encrypted when they are stored and it's required to retrieve the actual value of the cookies

**Comment by top31** [comment]
> The gadget has become like your dirty ass @aandergr

File: docs/codesnippets/import_chrome_session.py (hunk 1 above)
Comment (heno72) [reply docs/codesnippets/import_chrome_session.py:8]: I can't find a package for `win32crypt` in ubuntu 22.04

Comment (OMGhozlan) [reply docs/codesnippets/import_chrome_session.py:8]: I realized this issue and I'm working on a way of replacing it with PyCryptodome or at least making a check for Windows vs Linux

Comment (patilaum) [reply docs/codesnippets/import_chrome_session.py:8]: Hi
Any Update on the linux/ubunutu version?

Comment (patilaum) [reply docs/codesnippets/import_chrome_session.py:8]: @OMGhozlan

Comment (Niffecs) [reply docs/codesnippets/import_chrome_session.py:8]: @patilaum not realy :'(

**Pull request closed without merging**